keyboards/k34/info.json

```diff
@@ -0,0 +1,181 @@
+{
+    "keyboard_name": "k34", 
+    "url": "https://github.com/wongjingping/k34", 
+    "maintainer": "qmk", 
+    "layouts": {
+        "LAYOUT": {
+            "layout":[
+                {
+                    "label":"E",
+                    "x":2,
+                    "y":-0.1
```

Comment:
Key coordinates should be positive numbers. Please check this file is correct by pressing Ctrl+Shift+I in the Configurator.

Reply:
This is due to the unconventional layout of the keys it seems - let me know if this is a blocker that I should resolve.

Reply:
Yes, it doesn't look correct. You need to redo it without the rotation on the keys (I can see one of them has a rounding error) and place the sides together.
![image](https://user-images.githubusercontent.com/4781841/144735150-39e684d6-9a2c-44fc-8113-82521ed9aee0.png)

Use the Configurator to test the file, by pressing Ctrl+Shift+I. You can also make sure the keys are in the same order as the layout macro by turning on Fast Input.

Reply:
<img width="286" alt="Screen Shot 2021-12-05 at 10 20 35 PM" src="https://user-images.githubusercontent.com/4327467/144750441-8609266a-69a2-4942-b70b-09e1b598bd38.png">
Hmm I'm not sure why but it seems like the negative number is still there after I import it with ctrl-shift-I. Any tips for how to fix that?

Reply:
Assuming you generated it from the KLE, you have to fix that.

Reply:
changed it to 0 instead of a negative number - would that work?